### DISCUSSION TITLE
Support Jetbrains webstorm for IDE route

### DISCUSSION THREAD

**Discussion by yoni-noma** [Code-Forge-Net/Remix-Dev-Tools#116] [Ideas]
Hey, Currently the "open" button using the [VScodeButton](https://github.com/yoni-noma/Remix-Dev-Tools/blob/main/src/client/components/VScodeButton.tsx) component only support VSCode. but i believe that with minor changes it can support varius IDE's like webstorm. this button use the vite plugin for executing:
exec(`code -g "${normalizePath(source)}:${line}"`)
i guess that changing it to 
exec(`webstorm -g "${normalizePath(source)}:${line}"`)
or a settings context with the preffered IDE

**Comment by AlemTuzlak** [comment]
@yoni-noma this was implemented from v4.6.0 onwards!

**Reply by yoni-noma** [reply]
awesome thanks!

**Reply by yoni-noma** [reply]
@AlemTuzlak are you sure? i dont see a doc about it, and also i still only see the "Open in VSCode" button which doesn open my webstorm 🙏

**Reply by AlemTuzlak** [reply]
Yeah there's a doc on configuring your editor, you need to pass it a custom config to open it in webstorm and I think it's added as am example, let me try to find the link real quick

**Reply by AlemTuzlak** [reply]
@yoni-noma here you go https://remix-development-tools.fly.dev/docs/main/editor

**Reply by yoni-noma** [reply]
cool thanks!!
